index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>
+    Document
+  </title>
+</head>
+
+<body>
+  <div class=border>
+    <h1> Jaye Lin Resume </h1>
+    <div class="contact">
+      <p> <a href="Illusxia. carbonmade.com"> Portfolio </a><br>
+
+        <a href="Xiaoxxia23@Gmail.com"> Email </a> <br>
+        Baltimore, MD
+      </p>
+    </div>
```

Comment:
You can skip the `br` element if you treat this like a list instead:
```suggestion
    <ul class="contact">
      <li><a href="Illusxia. carbonmade.com"> Portfolio </a></li>
      <li><a href="Xiaoxxia23@Gmail.com"> Email </a></li>
      <li>Baltimore, MD</li>
    </ul>
```